tests/mock_crust/churn.rs

```diff
@@ -230,28 +231,28 @@ fn churn() {
 
         let mut expected_gets = ExpectedGets::default();
 
-        // Test messages from a node to itself, another node, a group and a section...
-        expected_gets.send_and_expect(data_id, auth_n0, auth_n0, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_n1, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_g0, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_s0, &mut nodes, min_group_size);
-        // ... and from a group to itself, another group, a section and a node...
-        expected_gets.send_and_expect(data_id, auth_g0, auth_g0, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_g1, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_s0, &mut nodes, min_group_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_n0, &mut nodes, min_group_size);
-        // ... and from a section to itself, another section, a group and a node...
+        // Test messages from a node to itself, another node, a section and a section...
```

Comment:
This should be still a "group" ("a section and a section" has a nice ring to it, though :grin:). Similarly below.